webclient/js/elems/asms.js

```diff
@@ -402,7 +402,7 @@ function buildAssessmentElements() {
                     $("#"+f.name).find("#AssessmentInfo").addClass("hidden");
 
                     // ENABLE ALL INPUTS IF ALL OF THOSE HAVE BEEN DISABLED FOR REVERSED PREVIOUSLY
-                    $("#"+f.name).find('input,button').not('input[name=BUD]').prop("disabled", false);
+                    // $("#"+f.name).find('input,button').not('input[name=BUD]').prop("disabled", false);
```

Comment:
Did you check that it does not keep BUD field enabled? Basically when you load any assessment which is not reversed, then you need to enable all fields except BUD. Right now, we don't allow to change the business for any instance from the form itself.